ceph.spec.in

```diff
@@ -1253,7 +1253,7 @@ done
 %endif
 
 %if 0%{?suse_version}
-%limit_build -m 2600
+%limit_build -m 3000
```

Comment:
Should this setting be moved to where we set `_smp_ncpus_max`?  Or if it's not possible because it has to be under `%build` or something, at least add a comment there explaining that OBS doesn't respect `_smp_ncpus_max` and a SUSE-specific `limit_build` is used instead and must be adjusted at the same time.

Otherwise, when someone bumps the memory requirement again they are almost guaranteed to miss `limit_build` because they are a thousand lines apart.

Reply:
@idryomov all makes sense to me, but i'd leave it to @smithfarm .

Reply:
i added a comment. hopefully it's more future proof.

Reply:
Just a nit, but I'd say "SUSE distros" instead of "suse_version".  Otherwise looks good to me, a comment should be enough.

Reply:
@idryomov updated and repushed.